src/Sources/LightPortal/Helper.php

```diff
@@ -56,7 +56,8 @@ public function session(?string $key = null): Session
 
 	public function hook(string $hook, array $vars = [], array $plugins = []): void
 	{
-		AddonHandler::getInstance()->run($hook, $vars, $plugins);
+		//AddonHandler::getInstance()->run($hook, $vars, $plugins);
```

Comment:
It's also used in these files:
- Themes/LightPortal/ViewPage.template.php
- Sources/LightPortal/Utils/Content.php
- Sources/LightPortal/Lists/PluginList.php